keyworker-specs/src/test/groovy/uk/gov/justice/digital/hmpps/keyworker/specs/MaintainAuthUsersSpecification.groovy

```diff
@@ -114,14 +114,13 @@ class MaintainAuthUsersSpecification extends BrowserReportingSpec {
         when: 'I select to add the vary role to the user'
         choose('Licence Vary')
 
-        then: 'I receive a role added message'
         at AuthUserPage
-        assert waitFor { messageBar.text() == 'Role Licence Vary added' }
 
+        then: 'I remove the GLOBAL_SEARCH role'
         oauthApi.stubAuthRemoveRole()
         roleRows[1].find("[data-qa='remove-button-GLOBAL_SEARCH']").click()
 
-        assert waitFor { messageBar.text() == 'Role Global Search removed' }
```

Comment:
not really testing anything here, would be better to check for the presence of the licence vary role and then the global search role being removed from the screen?